src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1088,6 +1090,172 @@ async def redirect_uri(request: Request):
+                        try:
+                            if config.tool_overrides is not None:
+                                for orig_name, override in config.tool_overrides.items():
+                                    if override.alias is not None:
+                                        alias_to_original[override.alias] = orig_name
+                        except Exception:
+                            pass
+
+                        # Create tool info list (always return configured tools; mark availability)
+                        tools_info: list[dict[str, Any]] = []
+                        available_count = 0
+                        for wf_fn, fn_short in zip(configured_full_to_fn.values(), configured_short_names):
```

Ruff (0.13.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)